assimilation_code/programs/restart_file_tool/restart_file_tool.rst

```diff
@@ -0,0 +1,8 @@
+PROGRAM ``restart_file_tool``
+=============================
+
+Overview
+--------
+
+This tool still exists in the "Classic" release of DART but is no longer needed in the "Manhattan" release. DART initial
+condition and restart files are now in NetCDF format and any standard NetCDF tool can be used to manipulate them.
```

Comment:
This file needs the boilerplate that states it is out-of-date with the warning/attention/banner common to all such files.

Reply:
@timhoar     add old banner

Reply:
done.